doc/configuration.rst

```diff
@@ -1333,6 +1361,14 @@ but more devices can be added by extending `on_activate()` in
 ``labgrid/driver/usbtmcdriver.py`` and writing a corresponding backend in
 ``labgrid/driver/usbtmc/``.
 
+FlashromDriver
+~~~~~~~~~~~~~~
+The :any:`FlashromDriver` is used to flash a rom, using the flashrom utility.
+Binds to:
+  flashrom_resource:
```

Comment:
CI complains about this line, looking at the entries above and below, `Binds to:` needs an empty newline before the list.